chat.ts

```diff
@@ -56,6 +58,7 @@ export interface ChatListOpts {
   cursor?: string;
   pageSize?: number;
   userEmail?: string | null;
+  externalUserId?: string | null;
```

Comment:
Oh yes this is copy pasta I guess. I copied the signature from the userEmail so that we could specify `null` to indicate "no external id", but I forgot to implement that on the API side.